portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,66 @@
   width: 650px;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+p.artist{
+    color:#5e7059;
+    line-height:30px;
+}
+.container{
```

Comment:
Here and below, add a new line between styles, just so it's easier to tell them apart